stages/org.osbuild.skopeo

```diff
@@ -22,10 +24,24 @@ def main(inputs, output, options):
             elif dest_type == "oci":
```

Comment:
Since skopeo (I think) creates the final dir, we could also merge all of them and always just create the parent, letting skopeo create the final component.